app/controllers/mechanic_rides_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MechanicRidesController < ApplicationController
+    def create
+        @mechanic = Mechanic.find(params[:mechanic_id])
+        if params[:ride_id] != nil
+            MechanicRide.create!(mechanic_id: params[:mechanic_id], ride_id: params[:ride_id])
```

Comment:
Hey there - this looks good; you may want to come back and add a sad path just for practice (not required for this IC though).